.gitignore

```diff
@@ -2,3 +2,4 @@
 *.iml
 target/*
 
+/target/
```

Comment:
Is that line necessary? Maybe line 3 only needs to have the trailing `/*` removed?